src/coreComponents/mesh/generators/InternalWellboreGenerator.cpp

```diff
@@ -185,7 +201,7 @@ void InternalWellboreGenerator::postProcessInput()
 
           // set the next radius a some combination of the inner and outer radius for this "element".
           constexpr real64 c = 0.5;
-          real64 const r_ip1 = m_radialCoords.back() + ( 1.0 - c ) * t_i + c * tElemSize_ip1_0;
+          real64 const r_ip1 = m_radialCoords.back() + m_autoSpaceRadialElems[iBlock] * ( ( 1.0 - c ) * t_i + c * tElemSize_ip1_0 );
```

Comment:
With the consideration of the multiplier m_autoSpaceRadialElems[iBlock], the calculation of the next radius becomes quite flexible and we still don't really need the averaging combination of the inner and outer radii, I would suggest to simplify this estimation to:

```suggestion
          real64 const r_ip1 = m_radialCoords.back() + m_autoSpaceRadialElems[iBlock] * t_i;
```